src/weather.py

```diff
@@ -0,0 +1,63 @@
+import os
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+OPEN_WEATHER_KEY = os.getenv('OPEN_WEATHER_KEY')
```

Comment:
Evitate le costanti scritte a mano. 
Ad esempio il nome della env var andrebbe recuperato dal file di configurazione (.env).
Se ci fossero più ambienti e le env var si chiamassero in modi diversi questo porterebbe problemi con la conf attuale!